Initialtask/config/services.yaml

```diff
@@ -4,6 +4,7 @@
 # Put parameters here that don't need to change on each machine where the app is deployed
 # https://symfony.com/doc/current/best_practices.html#use-parameters-for-application-configuration
 parameters:
+    filepath: '%kernel.project_dir%/public/stock.csv'
```

Comment:
define only path not file itself
Separate directory in public for imported files is preferable